packages/ts-api/src/handlers/example.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express';
```

Comment:
Also remove this? Or do we need it as a template for upcoming features?

Reply:
I was thinking of having the examples for future devs, but I suppose the files themselves will serve as that so we could remove them all